tv_apps/browser/js/browser_db.js

```diff
@@ -55,6 +40,7 @@ var BrowserDB = {
    * Populate topsites data from mozSettings to database.
    */
   initSingleVariant: function browserDB_initSingleVariant() {
+/* TODO: mozSettings to indexedDB
```

Comment:
If we are going to remove it, please just try to remove unused codes. If not and we don't want them to activate for now, please add some XXX to describe why we need to keep these codes.